### ISSUE TITLE
Jupyterlab file browser does not update file state after exporting a pipeline

### ISSUE THREAD

**Issue by akchinSTC** [elyra-ai/elyra#1387]
**Describe the issue**
Looks like Jupyterlab is caching the file state somewhere? Exported a pipeline in one format (kfp) ...open the file.....then exported to that same filename in a different format (airflow). File contents stays the same in the browser.

**To Reproduce**
1. Create or open a pipeline in the pipeline editor
2. Click on `export` and select KFP and then ok to create a KFP DSL *.py file
3. Open the new DSL file and check its contents
4. Click on `export` and this time select Airflow, then hit the checkbox for `ok to overwrite` and then ok to create a AA DAG *.py file, note that this uses the same filename and overwrites the existing KFP DSL file.
5. Open the exported. *.py file, still DSL, not an AA DAG

note that checking the file in the underlying file system via terminal (e.g. cat *.py) shows that the file was indeed overwritten and shows the correct AA DAG. 

**Screenshots or log output**

**Expected behavior**
Open up the "overwritten" file and see that the file was overwritten correctly with the new runtime pipeline info

**Deployment information**
Describe what you've deployed and how:
 - Elyra version: 2.1.0 rc1
 - Operating system: linux
 - Installation source: pypi
 - Deployment type: local , chrome browser

**Comment by kiersten-stokes** [comment]
Just wanted to chime in to say that I'm not able to reproduce on 2.1.0 rc2. The generated python file looks as it should (KFP vs Airflow) each time for me whether I close the file between exports or not.

**Comment by ptitzler** [comment]
I believe I did observe the general behavior in the past. What I don't remember is if I had removed the old file or not to work around it or not. 

Somewhat unrelated to the original issue I would like us to consider for export to generate a unique file name for each _runtime type_. 

Current behavior:
 Export generates a file `abc.py|yaml` for a pipeline named `abc` irrespective of what the export format is.
 
Cleaner behavior:
Export generates a file `abc_<XYZ_RUNTIME>.py|yaml` for a pipeline named `abc`. The file name now contains implicitly the information that the exported file is specific to the `XYZ` runtime environment.
